ui-admin/src/study/notifications/CreateTriggerModal.tsx

```diff
@@ -79,12 +83,12 @@ export default function CreateTriggerModal({ studyEnvContext, onDismiss, onCreat
 
   return <Modal show={true} className="modal-lg" onHide={onDismiss}>
     <Modal.Header closeButton>
-      <Modal.Title>Create Notification</Modal.Title>
+      <Modal.Title>Create Trigger</Modal.Title>
```

Comment:
```suggestion
      <Modal.Title>Create Automation</Modal.Title>
```
I feel like we've gotten a lot of negative feedback on the word trigger - maybe we should learn more into `Automation`?